web/src/repo/settings/RepoSettingsCodeIntelligencePage.tsx

```diff
@@ -0,0 +1,153 @@
+import { PageTitle } from '../../components/PageTitle'
+import { RouteComponentProps } from 'react-router'
+import { fetchLsifDumps } from './backend'
+import { FilteredConnection, FilteredConnectionQueryArgs } from '../../components/FilteredConnection'
+import { Timestamp } from '../../components/time/Timestamp'
+import { Link } from '../../../../shared/src/components/Link'
+import { switchMap, tap } from 'rxjs/operators'
+import { Subject, Subscription, Observable } from 'rxjs'
+import { LoadingSpinner } from '@sourcegraph/react-loading-spinner'
+
+interface LsifDumpNodeProps {
+    node: GQL.ILSIFDump
+}
+
+interface LsifDumpNodeState {}
+
+class LsifDumpNode extends React.PureComponent<LsifDumpNodeProps, LsifDumpNodeState> {
+    public state: LsifDumpNodeState = {}
+
+    public render(): JSX.Element | null {
+        return (
+            <div className="lsif-dump list-group-item">
+                <div className="lsif-dump__row lsif-dump__main">
+                    <div className="lsif-dump__meta">
+                        <div className="lsif-dump__meta-root">
+                            <Link to={this.props.node.projectRoot.url}>
+                                <strong>{this.props.node.projectRoot.path}</strong>
+                            </Link>{' '}
+                            <small className="text-muted lsif-dump__meta-commit">
+                                <code>{this.props.node.projectRoot.commit.abbreviatedOID}</code>
+                            </small>
+                        </div>
+                    </div>
+
+                    <small className="text-muted lsif-dump__meta-timestamp">
+                        <Timestamp noAbout={true} date={this.props.node.uploadedAt} />
+                    </small>
+                </div>
+            </div>
+        )
+    }
+}
+
+class FilteredLsifDumpsConnection extends FilteredConnection<{}, LsifDumpNodeProps> {}
+
+interface Props extends RouteComponentProps<any> {
+    repo: GQL.IRepository
+}
+
+interface State {
+    latestDumps?: GQL.ILSIFDump[]
+    loading: boolean
+    error?: Error
+}
+
+/**
+ * The repository settings code intelligence page.
+ */
+export class RepoSettingsCodeIntelligencePage extends React.PureComponent<Props, State> {
+    public state: State = { loading: true }
+
+    private updates = new Subject<void>()
+    private subscriptions = new Subscription()
+
+    public componentDidMount(): void {
+        eventLogger.logViewEvent('RepoSettingsCodeIntelligence')
+
+        this.subscriptions.add(
+            this.updates
+                .pipe(
+                    tap(() => this.setState({ loading: true })),
+                    switchMap(() => this.queryLatestDumps())
+                )
+                .subscribe(
+                    c =>
+                        this.setState({
+                            latestDumps: c.nodes.sort((a, b) => a.projectRoot.path.localeCompare(b.projectRoot.path)),
```

Comment:
With lodash's `sortBy`

```suggestion
                            latestDumps: sortBy(c.nodes, node => node.projectRoot.path),
```

Reply:
Neat!